### DISCUSSION TITLE
enable graphiql explorer

### DISCUSSION THREAD

**Discussion by riccardobarbatti** [99designs/gqlgen#2481] [Q&A]
Hello,

How do I enable explorer in graphiql playground template?

https://www.gatsbyjs.com/blog/2019-06-03-integrating-graphiql-explorer/

Is It possible?

Thanks

**Comment by alexantom** [comment]
this would be a good addition 
https://github.com/graphql/graphiql/pull/3374

https://deploy-preview-3393--graphiql-test.netlify.app/webpack/
